demoinfocs_test.go

```diff
@@ -151,7 +151,7 @@ func TestDemoInfoCs(t *testing.T) {
 			p.Cancel()
 			timer.Reset(time.Second * 1)
 			<-timer.C
-			t.Fatal("Parser locked up for more than one second after cancellation")
+			t.Fatal("parser locked up for more than one second after cancellation")
```

Comment:
Revert to upper-case

```suggestion
			t.Fatal("Parser locked up for more than one second after cancellation")
```